common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -1,241 +1,71 @@
 (function(define) {
     'use strict';
-// VideoAccessibleMenu module.
+// VideoTranscriptDownloadHandler module.
     define(
-'video/035_video_accessible_menu.js', [],
-function() {
+'video/035_video_accessible_menu.js', ['underscore'],
+function(_) {
     /**
      * Video Download Transcript control module.
      * @exports video/035_video_accessible_menu.js
      * @constructor
      * @param {jquery Element} element
      * @param {Object} options
      */
-    var VideoAccessibleMenu = function(element, options) {
-        if (!(this instanceof VideoAccessibleMenu)) {
-            return new VideoAccessibleMenu(element, options);
+    var VideoTranscriptDownloadHandler = function(element, options) {
+        if (!(this instanceof VideoTranscriptDownloadHandler)) {
+            return new VideoTranscriptDownloadHandler(element, options);
         }
 
-        _.bindAll(this, 'openMenu', 'openMenuHandler', 'closeMenu', 'closeMenuHandler', 'toggleMenuHandler',
-            'clickHandler', 'keyDownHandler', 'render', 'menuItemsLinksFocused', 'changeFileType', 'setValue'
-        );
+        _.bindAll(this, 'clickHandler');
 
         this.container = element;
         this.options = options || {};
 
-        if (this.container.find('.video-tracks')) {
+        if (this.container.find('.wrapper-downloads .wrapper-download-transcripts')) {
             this.initialize();
         }
+
+        return false;
     };
 
-    VideoAccessibleMenu.prototype = {
-        /** Initializes the module. */
+    VideoTranscriptDownloadHandler.prototype = {
+        // Initializes the module.
         initialize: function() {
             this.value = this.options.storage.getItem('transcript_download_format');
-            this.el = this.container.find('.video-tracks .a11y-menu-container');
-            this.render();
-            this.bindHandlers();
+            this.el = this.container.find('.list-download-transcripts');
+            this.clickHandler();
```

Comment:
Replace this with "this.el.on('click', '.btn-link', this.clickHandler);".

Executing "clickHandler" here is confusing.